Release notes for 1.14.5 (#8126)
dapr/dapr: docs/release_notes/v1.14.5.md

# Dapr 1.14.5

This update includes bug fixes:

> - [Fix content type metadata from query for outbox transaction](#fix-content-type-metadata-from-query-for-outbox-transaction)

Reviewer: ```suggestion
- [Fix content type metadata from query for outbox transactions](#fix-content-type-metadata-from-query-for-outbox-transaction)
```

Author: Done

> ## Fix content type metadata from query for outbox transaction

Reviewer: ```suggestion
## Fix content type metadata from query for outbox transactions
```

Author: Done

### Problem

> Transactions were not respecting the metadata in the transaction request.

Reviewer: ```suggestion
Outbox transactions were not respecting the metadata in the transaction request.
```

Author: Done

For example, content type specified in the metadata was not being applied and the message was being sent as plain text.

### Impact

> Outbox transactions were not respecting the metadata. 
> Cloudevents data was being sent as plain text and Cloudevents Middleware was not able to parse the data resulting in 415 error code.

Reviewer: ```suggestion
Outbox transactions were not respecting metadata. 
Cloudevents Middleware was unable to parse the metadata, resulting in 415 error codes.
```

Author: Done
